src/fs/class.ts

```diff
@@ -318,21 +326,33 @@ export class FileSystem {
     V = unknown,
   >(
     path: Path.File<PartitionedNonEmpty<Partition>> | { contentCID: CID } | { capsuleCID: CID } | {
-      capsuleRef: PrivateReference
+      capsuleKey: Uint8Array
     },
     dataType: D,
     options?: { offset: number; length: number }
   ): Promise<DataForType<D, V>>
   async read<V = unknown>(
     path: Path.File<PartitionedNonEmpty<Partition>> | { contentCID: CID } | { capsuleCID: CID } | {
-      capsuleRef: PrivateReference
+      capsuleKey: Uint8Array
     },
     dataType: DataType,
     options?: { offset: number; length: number }
   ): Promise<AnySupportedDataType<V>> {
     return this.#transactionContext().read<DataType, V>(path, dataType, options)
   }
 
+  /**
+   * Create a permalink to some public file system content.
+   * @group Querying
+   */
+  permalink(dataRoot: CID, path: Path.Distinctive<Path.Partitioned<Path.Partition>>): string {
+    if (this.#dependencies.depot.permalink) {
```

Comment:
note: when i started integrating the `ts-odd` `next` branch into collector this week I had to add this `babel` plugin to handle `#` methods:
```
@babel/plugin-transform-private-methods
```
This may just be a quirk of react-native(I haven't tried the latest updates in a svelte app yet), but perhaps it's something we want to highlight in the readme?

Reply:
Ah good point, maybe it’s because it’s a relatively new js feature (2021), ecmascript 2022 I think. The readme still needs to be rewritten but yeah we should definitely mention that. Can you change the compile target to es2022 in react native?

Reply:
looks like I can! It's strange, the app extends the expo config, which specifies a target of `ESNext`, which seems like it should have that functionality, but specifying `ES2022` explicitly seems to fix it 👌🏼

Reply:
Nice 👍 Yeah, esnext is confusing because it depends on when they last updated the pointer (could even be es2020 🤷). I'll mention in the readme that the SDK requires es2022.